llama.cpp/ggml-common.h

```diff
@@ -313,10 +325,11 @@ typedef struct {
 static_assert(sizeof(block_q6_K) == sizeof(ggml_half) + QK_K / 16 + 3*QK_K/4, "wrong q6_K block size/padding");
 
 // This is only used for intermediate quantization and dot products
+// [kawrakow] Note: I have switched the order of bsums and qs. This results in some performance gain on Arm
```

Comment:
As far as I can tell, we've never stored `q8_k` tensors inside gguf files. Can you confirm this change won't break any files?

Reply:
Yes, `Q8_K` is only used internally for quantized matrix multiplications and is never stored in a model file.